.eslintrc

```diff
@@ -96,7 +96,7 @@ rules:
   radix: 2
   vars-on-top: 0
   wrap-iife: 2
-  yoda: 0
+  yoda: [1, never]
```

Comment:
When there is a comparison Val < raw number < otherval it gives an error but that is really the best way to have that statement